go/libkb/account.go

```diff
@@ -119,8 +119,8 @@ func (a *Account) LoadLoginSession(emailOrUsername string) error {
 		return nil
 	}
 
-	ls := NewLoginSession(emailOrUsername, a.G())
-	if err := ls.Load(); err != nil {
+	ls := NewLoginSession(a.G(), emailOrUsername)
+	if err := ls.Load(NewMetaContextTODO(a.G())); err != nil {
```

Comment:
why is this NewMetaContextTODO?

Reply:
IT makes a `context.TODO()` since we don't have a context plumbed down. But we're going to get rid of `account.go` so I thought it was OK to leave this in.

Reply:
ok, will switch to that